MechJeb2/MechJebModuleAscentPVGSettingsMenu.cs

```diff
@@ -64,9 +66,9 @@ protected override void WindowGUI(int windowID)
 
                     GUILayout.BeginHorizontal();
                     GUILayout.Label($"{stats.KSPStage,3} {stats.DeltaV:##,###0} m/s");
-                    if (_ascentSettings.UnguidedStages.Contains(mjPhase))
+                    if (_ascentSettings.UnguidedStages.Contains(stats.KSPStage))
                         GUILayout.Label(" (unguided)");
-                    if (_ascentSettings.OptimizeStage == mjPhase)
+                    if (_ascentSettings.OptimizeStage == stats.KSPStage)
```

Comment:
this line might cause a problem if it matches multiple stages.  i don't know that it is fixable here, though.

really the fix that needs to happen is that the FFS needs the asymmetric burnout code added to it, in order to suppress the extra stages.